### DISCUSSION TITLE
How does replicache compare to Y.js

### DISCUSSION THREAD

**Discussion by daKmoR** [rocicorp/replicache#1001] [Q&A] [answered]
I'm following replicaches progress for some time and find the idea very intriguing but I never actually used it so far 🙈. (soo much to do - so little time)

And now I just stumbled over
https://github.com/yjs/yjs

and I was wondering how does it compare? it seems similar on the surface.

As "the surface" is all I know I am curious if anyone has more insight into what are the differences? 🙏

**Comment by prescod** [comment, marked as answer]
The difference with YJS is described here:

https://rocicorp.dev/blog/ready-player-two
